docs/configuration/migration/iaps-migration.md

```diff
@@ -0,0 +1,71 @@
+| **[Minimum Safety Threshold](../settings/therapy/units-limits.md#minimum-safety-threshold)** | decimal <br> (60 mg/dL) | Settings → Therapy → Units and Limits → Max IOB | **Threshold Setting** | decimal <br> (60) | Settings → Dynamic ISF → Threshold Setting |
+
+## Algorithm Settings
+
+### [Autosens](../settings/algorithm/autosens.md)
+
+| Trio Name | Setting Format <br> (example) | Location in App | iAPS Name | Setting Format <br> (example) | Location in App |
+|:---:|:---:|:---:|:---:|:---:|:---:|
+| [**Autosens Min**](../settings/algorithm/autosens.md#autosens-min) | percentage <br> (70%) | Settings → Algorithm → Autosens → Autosens Min | **Autosens Minimum** | decimal <br> (0.7) | Settings → OpenAPS → Autosens Minimum |
+| [**Autosens Max**](../settings/algorithm/autosens.md#autosens-max) | percentage <br> (120%) | Settings → Algorithm → Autosens → Autosens Max | **Autosens Maximum** | decimal <br> (1.2) | Settings → OpenAPS → Autosens Maximum |
+
+### [SMB (Super Micro Bolus)](../settings/algorithm/smb-settings.md)
+
+| Trio Name | Setting Format <br> (example) | Location in App | iAPS Name | Setting Format <br> (example) | Location in App |
+|:---:|:---:|:---:|:---:|:---:|:---:|
+| [**Enable SMB Always**](../settings/algorithm/smb-settings.md#enable-smb-always) | toggle <br> (On/Off) | Settings → Algorithm → Super Micro Bolus (SMB) → Enable SMB Always | **Enable SMB Always** | toggle <br> (On/Off) | Settings → OpenAPS → Enable SMB Always |
+| [**Allow SMB with High Temp Target**](../settings/algorithm/smb-settings.md#allow-smb-with-high-temptarget) | toggle <br> (On/Off) | Settings → Algorithm → Super Micro Bolus (SMB) → Allow SMB With High Temptarget | **Allow SMB With High Temptarget** | toggle <br> (On/Off) | Settings → OpenAPS → Allow SMB With High Temptarget |
+| [**Enable UAM**](../settings/algorithm/smb-settings.md#enable-uam) | toggle <br> (On/Off) | Settings → Algorithm → Super Micro Bolus (SMB) → Enable UAM | **Enable UAM** | toggle <br> (On/Off) | Settings → OpenAPS → Enable UAM |
+| [**Max SMB Basal Minutes**](settings/algorithm/smb-settings.md#max-smb-basal-minutes) | decimal <br> (30 min) | Settings → Algorithm → Super Micro Bolus (SMB) → Max SMB Basal Minutes | **Enable UAM** | decimal <br> (30) | Settings → OpenAPS → Max SMB Basal Minutes |
+| [**Max UAM Basal Minutes**](settings/algorithm/smb-settings.md#max-uam-basal-minutes) | decimal <br> (30 min) | Settings → Algorithm → Super Micro Bolus (SMB) → Max UAM Basal Minutes | **Max UAM SMB Basal Minutes** | decimal <br> (30) | Settings → OpenAPS → Max UAM SMB Basal Minutes |
+| [**Max Delta-BG Threshold SMB**](settings/algorithm/smb-settings.md#max-delta-bg-threshold-smb) | percentage <br> (20%) | Settings → Algorithm → Super Micro Bolus (SMB) → Max Delta-BG Threshold SMB | **Max Delta-BG Threshold SMB** | decimal <br> (0.2) | Settings → OpenAPS → Max Delta-BG Threshold SMB |
```

Comment:
I think for Trio, this is called Max Allowed Glucose Rise for SMB, Trio default is 20%